### PR TITLE
Update enabling-codespaces-for-your-organization.md

### PR DESCRIPTION
**Pull request by highfi13g**
### Why:

Need to be more verbose about the requirements. CODESPACES is currently not available for orgs with IP Allow list enabled.

### What's being changed:

The current documentation  does not state the need to have IP Allow list disabled in order for Codespaces to be visible. I added a line noting this as a requirement to use codespaces within an organization.

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@highfi13g 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by pedramgoli — COMMENTED** [review]
content/codespaces/managing-codespaces-for-your-organization/enabling-codespaces-for-your-organization.md

File: content/codespaces/managing-codespaces-for-your-organization/enabling-codespaces-for-your-organization.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ To use codespaces in your organization, you must do the following:
 - Ensure that users have [at least write access](/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization) to the repositories where they want to use a codespace. 
 - [Enable {% data variables.product.prodname_codespaces %} for users in your organization](#configuring-which-users-in-your-organization-can-use-codespaces). You can choose allow {% data variables.product.prodname_codespaces %} for selected users or only for specific users.
 - [Set a spending limit](/billing/managing-billing-for-github-codespaces/managing-spending-limits-for-codespaces)
+- Ensure your organization does not have [IP Allow list](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-allowed-ip-addresses-for-your-organization) enabled.
```
Comment (hubwriter) [inline content/codespaces/managing-codespaces-for-your-organization/enabling-codespaces-for-your-organization.md:29]: Slight tweak to the wording to match our style guide:
```suggestion
- Ensure that your organization does not have an IP address allow list enabled. For more information, see "[Managing allowed IP addresses for your organization](/organizations/keeping-your-organization-secure/managing-allowed-ip-addresses-for-your-organization)."
```

**Review by hubwriter — DISMISSED** [review]
@highfi13g - Many thanks for making this change. 🙏 

I've made a small change, which I'm going to go ahead an apply so that I can get this merged.

Thanks again

**Review by hubwriter — APPROVED** [review]

**Pull request merged**